frontend/src/pages/CreateRecording.tsx

```diff
@@ -1,12 +1,45 @@
 import {EuiButton, EuiPageTemplate, EuiText} from '@elastic/eui';
-import {MutableRefObject, Ref, useRef, useState} from "react";
+import {MutableRefObject, Ref, useCallback, useEffect, useRef, useState} from "react";
 import {EuiFieldText, EuiFlexGroup, EuiFlexItem} from '@elastic/eui';
-import {EuiInnerText} from "@elastic/eui";
+import React, { Component } from 'react'
+import Terminal from 'react-console-emulator'
 
 function CreateRecordingPage() {
-  const nameRef = useRef(null)
-  const volumeRef = useRef(null)
-  const programRef = useRef(null)
+  const [name, setName] = useState('');
+  const [volume, setVolume] = useState('');
+  const [program, setProgram] = useState('');
+  const [commands, setCommands] = useState('');
+  const terminal = React.createRef()
+
+  const sendAPICall = useCallback(function (name:string, volume:string, program:string) {
```

Comment:
You don't need the parameters in these functions as you're using the variables from the upper scope